src/app/cli/src/init/client.ml

```diff
@@ -683,6 +683,17 @@ let snark_job_list =
          | Error e ->
              print_rpc_error e ))
 
+let snark_pool_list =
+  let open Deferred.Let_syntax in
+  let open Command.Param in
+  Command.async ~summary:"List of snark works in the snark pool in JSON format"
+    (Cli_lib.Background_daemon.init (return ()) ~f:(fun port () ->
+         match%map dispatch Daemon_rpcs.Snark_pool_list.rpc () port with
+         | Ok str ->
+             printf "%s" str
```

Comment:
I cringe every time I see a `printf "%s" s` as opposed to a `Stdout.write s` or similar.